### PR TITLE
Login components

### PR DESCRIPTION
**Pull request by AbhiGattineni**

### CODE REVIEW

File: src/Components/Icon/Icon.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { Facebook, Twitter, Google, GitHub } from "@mui/icons-material";
+
+import { SvgIconProps } from "@mui/material";
+
+type SocialMediaIconProps = {
+  type: string;
+} & SvgIconProps;
+
+const Icon = ({ type, ...props }: SocialMediaIconProps) => {
```
Comment (RushilBasappa) [inline src/Components/Icon/Icon.tsx]: Rename this component to SocialMediaIcon.

File: src/Components/SocialMediaButton/SocialMediaButton.tsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { Button, ButtonProps } from "@mui/material";
```
Comment (RushilBasappa) [inline src/Components/SocialMediaButton/SocialMediaButton.tsx]: This should be in Button folder

**Review by RushilBasappa — CHANGES_REQUESTED** [review]

File: src/Components/Icon/Icon.tsx (hunk 1 above)
Comment (AbhiGattineni) [reply src/Components/Icon/Icon.tsx]: what if you want to use just icon?

File: src/Components/SocialMediaButton/SocialMediaButton.tsx (hunk 2 above)
Comment (AbhiGattineni) [reply src/Components/SocialMediaButton/SocialMediaButton.tsx]: changed

File: src/Components/Icon/Icon.tsx (hunk 1 above)
Comment (RushilBasappa) [reply src/Components/Icon/Icon.tsx]: SocialMediaIcon is a type of an Icon component. It should reside in Icon folder but with a different name.
Icon component is separate component

File: src/App.tsx [hunk 3]
```diff
@@ -1,28 +1,48 @@
...
 import { Button } from "Components/Button";
+import { SocialMediaButton } from "Components/Button";
 
 function App() {
-  const [count, setCount] = useState(0);
-
   return (
     <>
-      <h1>React</h1>
+      <Box sx={{ display: "flex", alignItems: "center" }}>
```
Comment (RushilBasappa) [inline src/App.tsx:14]: This is login related.
Create a new folder named Pages and move this code under Login.tsx and import it here

File: src/Components/Button/index.tsx [hunk 4]
```diff
@@ -1,3 +1,4 @@
 import Button from "./Button";
+import SocialMediaButton from "./SocialMediaButton";
 
-export { Button };
+export { Button, SocialMediaButton };
```
Comment (RushilBasappa) [inline src/Components/Button/index.tsx:4]: Perfect!!

**Review by RushilBasappa — CHANGES_REQUESTED** [review]

File: src/App.tsx (hunk 3 above)
Comment (AbhiGattineni) [reply src/App.tsx:14]: I just created the components there for testing, I am currently working on that . You can ignore this for now.

**Pull request closed without merging**